.circleci/parallel_cli_tests.py

```diff
@@ -37,10 +37,13 @@
 else:
     filters = list(selected_tests)
 
-subprocess.run(
-    ['test/cmdlineTests.sh'] + filters,
-    stdin=sys.stdin,
-    stdout=sys.stdout,
-    stderr=sys.stderr,
-    check=True,
-)
+try:
+    subprocess.run(
+        ['test/cmdlineTests.sh'] + filters,
+        stdin=sys.stdin,
+        stdout=sys.stdout,
+        stderr=sys.stderr,
+        check=True,
+    )
+except subprocess.CalledProcessError as exception:
+    sys.exit(exception)
```

Comment:
I didn't know you could exit with an object instead of an integer code. Neat.

Reply:
I actually didn't know that myself. I forgot a string conversion here at first, then realized it still works :)

But I did know that you can pass in a string and it makes it exit with code 1 and print the message to stderr, which is a nice shortcut.